src/parsers/parseSchema.ts

```diff
@@ -38,7 +38,7 @@ const addMeta = (schema: JSONSchema7, parsed: string): string => {
 };
 
 const addDefaults = (schema: JSONSchema7, parsed: string): string => {
-  if (schema.default) {
+  if ('default' in schema) {
```

Comment:
Please use `!== undefined` so we avoid differences between pre serialised data vs "raw" json (which won't ever have `undefined` in a key)

Reply:
Just to clarify, the JSON equivalence of 
```javascript
const data = {
  type: "string",
  default: undefined
}
```

Would just be
```json
{
  "type": "string"
}
```

With `in`, the JS data would produce `.default(undefined)` but the JSON wouldn't.

Reply:
Makes sense